example/logic/paper.md

```diff
@@ -40,10 +40,14 @@ affiliations:
 date: 3 March 2020
 bibliography: paper.bib
 event: Fukuoka2019
+biohackathon_name: "NBDC/DBCLS BioHackathon - Other"
```

Comment:
this may not be 'Other'?

Reply:
Ah okay this is just an example for the other hackathon. Fine but a bit confusing since Fukuoka's one is not one of the others?

Reply:
I just needed to make sure 'Other' worked. Should remove this now :)